seqr/utils/search/hail_search_utils.py

```diff
@@ -28,10 +28,14 @@ def get_hail_variants(samples, search, user, previous_search_results, genome_ver
     end_offset = num_results * page
     search_body = _format_search_body(samples, genome_version, user, end_offset, search)
 
+    frequencies = search_body.pop('freqs', None)
+    if frequencies and frequencies.get('callset'):
+        frequencies['seqr'] = frequencies.pop('callset')
```

Comment:
It seems to me that `seqr` here means `seqr callset`. Is it better to use `seqrCallset` or `seqrPop`?

Reply:
No, that would be an anti-pattern.  This is nested under a `freqs` dictionary in the body, so its very explicit that its rteferring to a frequency, and we don't  use `gnomad_genomes_pop` or the like for the other populations